routes/api-v0.1/club-route.js

```diff
@@ -66,8 +106,7 @@ module.exports = function(app) {
     clubRoute = new ClubRoute(app);
     app.get(route.ClubsRoute, clubRoute.getClubs);
     app.get(route.ClubRoute, clubRoute.getClub);
-    //app.put(route.ClubRoute, clubRoute.updateClub);
+    app.put(route.ClubRoute, clubRoute.updateClub);
     app.post(route.ClubsRoute, clubRoute.saveClub);
-    //app.post(route.ClubUploadImageRoute, clubRoute.uploadClubImage);
-    //app["delete"](route.ClubRoute, clubRoute.removeClub);
+    app["delete"](route.ClubRoute, clubRoute.removeClub);
```

Comment:
This way to use the API to remove(app["delete"]) is deprecated, because we are using the parses to un-encoded the request as extended. You better change the API as app.delete(route.ClubRoute, clubRoute.removeClub);. Thanks